draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -654,476 +526,437 @@
+	so dCDNs are free to assign whatever structure they desire to the URIs for CI/T resources. 
+	Therefore, uCDNs MUST NOT make any assumptions regarding the structure of CI/T URIs or the mapping between CI/T
+	objects and their associated URIs. The URIs used in the examples in this document are purely illustrative and are not 
+	intended to impose a definitive structure on CI/T interface implementations.
+    </t>
+
+    <section title="Creating Triggers" anchor="creating-triggers">
+    <t>
+   	To create a new trigger, uCDN makes an HTTP POST request with the trigger representation to the root interface URL.
+	The trigger representation MUST include the mandatory attributes of <xref target="ci-trigger-resource">trigger resource</xref>.
    </t>
    <t>
-   All dCDNs implementing CI/T MUST support the HTTP GET, HEAD, POST,
-   and DELETE methods as defined in <xref target="RFC9110"/>.</t>
+	uCDN MAY also optionally specify optional trigger v2 specification attributes, namely trigger labels and trigger extensions, 
+	as well as the optional "cdn-path" attribute of the trigger resource. 
+   </t>
 
    <t>
-   The only representation specified in this document is JSON <xref target="RFC8259"/>.
-   It MUST be supported by uCDN and by dCDN.</t>
+	dCDN validates the trigger resource representation sent by uCDN. If the representation is malformed or uCDN does not have 
+	sufficient access rights, dCDN MUST either respond with an appropriate 4xx HTTP error code and not create a
+	trigger resource or create a trigger resource with "failed" state and an appropriate <xref target="error-v2-description">
+	Error.v2 Description</xref>.
+   </t>
 
    <t>
-   The URL of dCDN's collection of all Trigger Status Resources
-   needs to be either discovered by or configured in uCDN. The
-   mechanism for discovery of that URL is outside the scope of this
-   document.</t>
+	The new trigger resource is created in "pending" state. 
+	If successful, The HTTP response to the uCDN trigger creation request MUST have status code 201 (Accepted) and MUST convey the 
+	URI of the newly created trigger resources in the Location response header field <xref target="RFC9110"/>. 
+	The HTTP response SHOULD include the updated representation of the trigger resource. 
+   	This is particularly important in cases where dCDN processed the trigger immediately.</t>
 
    <t>
-   CI/T Commands are POSTed to dCDN's collection of all Trigger
-   Status Resources. If a CI/T Trigger Command is accepted by dCDN,
-   dCDN creates a new Trigger Status Resource and returns its URI to
-   uCDN in an HTTP 201 response. The triggered activity can then be
-   monitored by uCDN using that resource and the collections
-   described in <xref target="collections-of-trigger-status-resources"/>.</t>
+	Once a trigger resource has been created, dCDN MUST NOT reuse its URI, even after the trigger resource has been fully removed. 
+	It it therefore recommended for dCDN to utilize unique UUID identifiers. 
```

Comment:
s/It it/It is/

Reply:
Also, wording of this sentence sounds awkward.  Maybe, "It is therefore recommended that dCDN utilize unique UUID identifiers."

Reply:
fixed. 
also added a reference to RFC9562 https://datatracker.ietf.org/doc/html/rfc9562